### PR TITLE
Xml to src

### PR DESCRIPTION
**Pull request by zeroadam**
Move Xml to Friendica\Util namespace

Related to #3878

### CODE REVIEW

**Comment by zeroadam** [comment]
Searched `addon` but found no references.

**Comment by MrPetovan** [comment]
As with `DBM` and `DFRN`, shouldn’t it be `XML` instead of `Xml`?

**Comment by zeroadam** [comment]
Yes, I thought as much as well. When I started I was looking at the spreadsheet and it was using this capitalization. I'll make the change!

**Comment by MrPetovan** [comment]
I understand, the spreadsheet is a collaborative document and doesn’t always reflect the latest consensus until someone changes it. Please don’t assume it is necessary right.

File: include/api.php [hunk 1]
```diff
@@ -42,647 +42,658 @@
 $API = array();
 $called_api = null;
 
-/// @TODO Fix intending
```
Comment (MrPetovan) [inline include/api.php]: Nice! 😄

Comment (zeroadam) [reply include/api.php]: I knew you would love this!!

Comment (MrPetovan) [reply include/api.php]: You removed a TODO and a typo in one fell swoop! 😂

**Comment by zeroadam** [comment]
🤣

**Comment by annando** [comment]
Have you checked the API as well? There the changes are too much to see if it should work.

**Comment by zeroadam** [comment]
Yes, so far as to use twidere and see successful GET and such in the logs

**Pull request merged**